config/database.yml

```diff
@@ -58,6 +57,11 @@ development:
 test:
   <<: *default
   database: vglist_test
+  # username: postgres
+  
+  # password: <%= ENV['VGLIST_DATABASE_PASSWORD'] || 'password' %>
+  # host: <%= ENV['VGLIST_DATABASE_HOST'] || 'localhost' %>
```

Comment:
These seem unnecessary?

Reply:
I think line 63 is likely unnecessary yeah.